### PR TITLE
(RE-8455) Remove all references to PC1 in GPG file names

### PR DESCRIPTION
**Pull request by mwaggett**
We decided to use our existing GPG key for the time being, rather than creating a new one.
This commit just renames the GPG destination files to not include 'PC1'.

### CODE REVIEW

**Comment by mwaggett** [comment]
@melissa I kept it as RE-8455 to document our decision wrt that ticket.

File: configs/components/gpg_key.rb [hunk 1]
```diff
@@ -3,14 +3,14 @@
 
   if platform.is_deb?
     pkg.add_source 'file://files/puppetlabs-keyring.gpg'
-    pkg.install_file 'puppetlabs-keyring.gpg', '/etc/apt/trusted.gpg.d/puppetlabs-pc1-keyring.gpg'
+    pkg.install_file 'puppetlabs-keyring.gpg', '/etc/apt/trusted.gpg.d/puppetlabs-keyring.gpg'
```
Comment (underscorgan) [inline configs/components/gpg_key.rb]: Can this be puppet-keyring? I think puppetlabs-keyring will conflict with the old puppetlabs release packages

File: configs/components/gpg_key.rb [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
   else
     pkg.add_source("file://files/RPM-GPG-KEY-puppetlabs.gpg")
     pkg.add_source("file://files/RPM-GPG-KEY-puppet.asc")
     pkg.add_source("file://files/RPM-GPG-KEY-nightly-puppetlabs")
-    pkg.install_file 'RPM-GPG-KEY-puppetlabs.gpg', '/etc/pki/rpm-gpg/RPM-GPG-KEY-puppetlabs-PC1'
-    pkg.install_file 'RPM-GPG-KEY-puppet.asc', '/etc/pki/rpm-gpg/RPM-GPG-KEY-puppet-PC1'
-    pkg.install_file 'RPM-GPG-KEY-nightly-puppetlabs', '/etc/pki/rpm-gpg/RPM-GPG-KEY-nightly-puppetlabs-PC1'
+    pkg.install_file 'RPM-GPG-KEY-puppetlabs.gpg', '/etc/pki/rpm-gpg/RPM-GPG-KEY-puppetlabs'
+    pkg.install_file 'RPM-GPG-KEY-puppet.asc', '/etc/pki/rpm-gpg/RPM-GPG-KEY-puppet'
+    pkg.install_file 'RPM-GPG-KEY-nightly-puppetlabs', '/etc/pki/rpm-gpg/RPM-GPG-KEY-nightly-puppetlabs'
```
Comment (underscorgan) [inline configs/components/gpg_key.rb]: These will also all need something appended like PC1 so we don't conflict with old release packages

Comment (mwaggett) [inline configs/components/gpg_key.rb]: Is there a reason that the destination files don't have extensions while the source ones do (lines 11 and 12)?

Comment (mwaggett) [inline configs/components/gpg_key.rb]: ^ @underscorgan

Comment (underscorgan) [inline configs/components/gpg_key.rb]: I feel like there was, and it seems to be standard for the RPM keys to not have extensions, though I'm not sure why the sources have names with extensions. I feel like I'm forgetting somethign though, @melissa @mckern @shrug do you know?

Comment (mwaggett) [inline configs/components/gpg_key.rb]: I'm trying to figure out the difference between `RPM-GPG-KEY-puppetlabs.gpg` and `RPM-GPG-KEY-puppet.asc` so I can rename them appropriately.

Comment (underscorgan) [inline configs/components/gpg_key.rb]: @mwaggett conversation https://github.com/puppetlabs/puppetlabs-release/pull/64 might help

Comment (underscorgan) [inline configs/components/gpg_key.rb]: They are both gpg keys formatted the same so I don't know why they have different extensions, the main difference is they each have different keys (one hte old key, one the current). It's mostly just the target that can't be the same as in other release packages.

**Comment by mwaggett** [comment]
@underscorgan I replaced 'PC1' with 'release'.

**Pull request merged**